aiorwlock/__init__.py

```diff
@@ -165,12 +165,17 @@ def _release(self, lock_type):
         self._wake_up()
 
     def _wake_up(self):
+        """If no one is reading or writing, wake up write waiters
```

Comment:
This is not a docstring, but a comment. Ref: PEP 257.

Reply:
Agree. Docstrings should be applied to public API only, `_wake_up` starts from underscore.

Reply:
There's nothing wrong with a private method having a docstring, I just wanted it to comply standard: short sentence with trailing doc + paragraph after a space.

Reply:
And other thing is that it was an implementation explanation, while docstring is about different things.

Reply:
Yes. You are right.